packages/core/examples/colorbar/index.ts

```diff
@@ -0,0 +1,575 @@
+
+    volumes.push({
+      volumeId: ptVolumeId,
+      callback: setPetColorMapTransferFunctionForVolumeActor,
+    });
+  }
+
+  // Set the volume on the viewport
+  await viewport.setVolumes(volumes);
+
+  // Update the colormap to the active one on PT/CT viewports
+  if (fusion) {
+    setPTViewportColormap(viewportInfo, currentPTColormapName);
+  }
+}
+
+// Creates one or more containers at the right side of the viewport
+function createRightColorBarContainers(numContainers) {
+  const containers = [];
+  const height = 100 / numContainers;
+  let top = 0;
+
+  for (let i = 0; i < numContainers; i++, top += height) {
+    const container = document.createElement('div');
+
+    Object.assign(container.style, {
+      position: 'absolute',
+      top: `${top}%`,
+      left: `calc(100% - ${colorBarWidth}px)`,
+      width: `${colorBarWidth}px`,
+      height: `${100 / numContainers}%`,
+    });
+
+    containers.push(container);
+  }
+
+  return containers;
+}
+
+// Creates one or more containers at the bottom of the viewport
+function createBottomColorBarContainers(numContainers) {
+  const containers = [];
+  const width = 100 / numContainers;
+  let left = 0;
+
+  for (let i = 0; i < numContainers; i++, left += width) {
+    const container = document.createElement('div');
+
+    Object.assign(container.style, {
+      position: 'absolute',
+      top: `calc(100% - ${colorBarWidth}px)`,
+      left: `${left}%`,
+      width: `${width}%`,
+      height: `${colorBarWidth}px`,
+    });
+
+    containers.push(container);
+  }
+
+  return containers;
+}
+
+// Creates one or more containers at the right side or at the bottom
+// of the viewport based on `colorBar.position` config
+function initializeColorBarContainers(viewportInfo, viewportContainer) {
+  const numContainers = viewportInfo.fusion ? 2 : 1;
+  const containers =
+    viewportInfo.colorBar?.position === 'right'
+      ? createRightColorBarContainers(numContainers)
+      : createBottomColorBarContainers(numContainers);
+
+  containers.forEach((container) => {
+    Object.assign(container.style, {
+      boxSizing: 'border-box',
+      display: 'block',
+      border: 'solid 1px #555',
+      cursor: 'initial',
+    });
+
+    viewportContainer.appendChild(container);
+  });
+
+  return containers;
+}
+
+// Create instaces of the color bars for CT or PT/CT viewports and add them to the DOM
+function initializeColorBars(viewportInfo, colorBarContainers) {
+  const { fusion, volumeIds = [], colorBar, viewportInput } = viewportInfo;
+  const { element } = viewportInput;
+
+  const scaleStyle = {
+    font: '12px Arial',
+    color: '#fff',
+    maxNumTicks: 8,
+    tickSize: 5,
+    tickWidth: 1,
+    labelMargin: 3,
+  };
+
+  const ctColorBar = new ViewportColorBar({
+    id: 'ctColorBar',
+    element,
+    container: colorBarContainers[0],
+    volumeId: volumeIds[0],
+    colormaps,
+    activeColormapName: 'Grayscale',
+    rangeTextPosition: ColorBarRangeTextPosition.TopOrLeft,
```

Comment:
I'm confused on why we have TopOrLeft? Can't we just divide them in top, bottom, left, right? and just throw for those that are not implemented?